torch/onnx/_internal/fx/passes/type_promotion.py

```diff
@@ -55,6 +67,16 @@ class TypePromotionSnapshot:
     """Expected output dtype of the node."""
 
 
+@_beartype.beartype
+def _get_fake_tensor_val(node: torch.fx.Node) -> fake_tensor.FakeTensor:
```

Comment:
Add a brief on how it should be used?

Reply:
syntactic sugar, maybe call it `_fake_tensor_from_meta`?

Reply:
or `_fake_tensor_from_node_meta`